while-true-do/ansible-role-cockpit
# Rework: Refactoring for features and structure

- Add: meta files added
- Update: LICENSE
- Add: docs added
- Add: tests added
- Remove: old tests dropped
- Add: update-meta-files.sh added
- Remove: obsolete vars/main.yml dropped
- Rework: Variables renamed to proper format
- Add: new packages added like sosreport
- Add: Dependencies maintained
- Add: Configuration options for cockpit add
- Add: Templates for config added

- Resolves: #2 #3 #4 
- See also: #1 is not resolved and needs extra work
